src/Faker/Provider/de_DE/Company.php

```diff
@@ -11,5 +11,14 @@ class Company extends \Faker\Provider\Company
         '{{lastName}}',
     );
 
+    /**
+     * @link http://www.personalseite.de/information/titel.htm
+     */
+    protected static $jobTitleFormat = array(
+        'Abteilungsdirektor', 'Arbeitsdirektor', 'Aufsichtsrat', 'Beirat', 'Bereichsleiter', 'Betriebsleiter', 'Finanzvorstand', 'Geschäftsführender Gesellschafter', 'Geschäftsführer', 'Gesellschafter',
```

Comment:
Apart from possible copyright issues, can you trim the job titles, please? There are a few entries with trailing whitespace.

Reply:
Hi, thanks for your Feedback, but the Job titles including a whitespace does not make sense without the whitespace. Therefore need to be removed completly, but what is the reason that whitespace is not allowed?